x-pack/plugins/uptime/server/lib/saved_objects/saved_objects.ts

```diff
@@ -34,6 +34,42 @@ export const registerUptimeSavedObjects = (
       type: syntheticsServiceApiKey.name,
       attributesToEncrypt: new Set(['apiKey']),
     });
+    encryptedSavedObjects.registerType({
+      type: syntheticsMonitor.name,
+      attributesToEncrypt: new Set([
```

Comment:
I've encrypted not all but nearly ever user-provided value, anything that is a string and not a value from a select box or that can be configured custom by the customer.